### DISCUSSION TITLE
Unable to run feed create API on postman

### DISCUSSION THREAD

**Discussion by paritshivani** [Sunbird-Lern/Community#207] [Issues]
Hello team,
hope you are doing well

We have one requirement where we need to create some custom feed notifications. For this I was using following API but getting server error
Sharing curl and response here. Please help us to understand exact issue

API:
`curl --location '{{host}}/api/user/feed/v1/create' \
--header 'Content-Type: text/plain' \
--header 'Authorization: Bearer -----------' \
--header 'x-authenticated-user-token: ----------' \
--data '{
  "request": {
     "userId": "5d757783-a86a-40cd-a814-1b6a16d37cb6",
        "contentId": "do_1141280160518635521128",
        "contentType": "Course",
        "priority": 1,
        "category": "Interests",
        "visibility": [
            "Default"
        ]
  }
}'`

Response: 
`{
    "id": "api.user.feed.create",
    "ver": "v1",
    "ts": "2024-09-27 13:20:08:579+0000",
    "params": {
        "resmsgid": null,
        "msgid": null,
        "err": "UOS_0069",
        "status": "FAILED",
        "errmsg": "server error"
    },
    "responseCode": "serverError",
    "result": {}
}`

@komalm, @AmiableAnil , @amitpriyadarshi, @reshmi-nair , @pallakartheekreddy, @HarishGangula, @swayangjit @vinukumar-vs @rajeevsathish

**Comment by paritshivani** [comment]
Hello @pallakartheekreddy, @HarishGangula, @swayangjit @vinukumar-vs @rajeevsathish

Can we get guidance here Please??

**Reply by HarishGangula** [reply]
@maheshkumargangula @gayathrisrividya Please check

**Reply by paritshivani** [reply]
Hello falks,
Any updates?

**Reply by GayathriSrividya** [reply]
Hi @paritshivani , 
I’ve reviewed the request body you provided for creating custom feed notifications, and it appears to be structured correctly. However, the API response has limited information for us to debug.

To assist you further, could you please share the logs from the userorg-service and notification-service? This will help us identify the root cause of the issue.
